book/website/ethical-research/law-policy.md

```diff
@@ -0,0 +1,263 @@
+# Ethics and Law, Policy and Human Rights
+
+## Prerequisites / recommended skill level
+None
+
+## Summary
+Law, human rights and policy all relate to ethics, but they are not the same thing.
+
+You should be aware of the laws and policies that apply to you and your work.
+However, this is not sufficient to ensure that your work is ethical. 
+
+Human rights provide a legal framework for protecting individuals. 
```

Comment:
It can be challenged that human rights provide legal frameworks at all.

Reply:
What do you mean? International human rights law is a legal framework that is enforceable at international levels. It's very slow and not particularly accessible, but it is a system of law.